src/main/java/pages/toolsqapages/AlertsPage.java

```diff
@@ -0,0 +1,92 @@
+package pages.toolsqapages;
+
+import org.apache.log4j.Logger;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import pages.BasePage;
+
+public class AlertsPage extends BasePage {
+
+    @FindBy(id = "alertButton")
+    WebElement alertButton;
+
+    @FindBy(id = "timerAlertButton")
+    WebElement timerAlertButton;
+
+    @FindBy(id = "confirmButton")
+    WebElement confirmBoxButton;
+
+    @FindBy(id = "promtButton")
+    WebElement promptButton;
+
+    @FindBy(xpath = "//span[@id='confirmResult']")
+    WebElement confirmResult;
+
+    @FindBy(xpath = "//span[@id='promptResult']")
+    WebElement promptResult;
+
+    private static final Logger LOG = Logger.getLogger(AlertsPage.class);
+
+    public AlertsPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public String getConfirmationResultText() {
+        return confirmResult.getText();
+    }
+
+    public String getPromptResultText() {
+        return promptResult.getText();
+    }
+
+    public void handleInformationAlert() {
+        LOG.info("Click button to information alert");
+        alertButton.click();
+        Alert alert = driver.switchTo().alert();
```

Comment:
add class for work with Alert, to not duplicate code

Reply:
done